usage_report/utils/trackingprotection.py

```diff
@@ -61,3 +61,62 @@ def get_number_of_users(df, count_name):
                          'country',
                          F.coalesce('pct_TP', F.lit(0)).alias('pct_TP'))
     return join_df
+
+
+def pct_etp(data,
+            date,
+            period=7,
+            country_list=None):
+    """ Calculate proportion of users in WAU that have a
+        extended tracking protection = on for a session (at least 1)
+        Parameters:
+        data: spark df, main summary
+        date: string, with the format 'yyyyMMdd'
+        period: int, period to check proportion for, 7 for WAU
+        country_list: a list of country names in string
+
+        Returns:
+        a spark df with the following columns
+        - columns: | submission_date_s3 | country | pct_ETP |
+    """
+    enddate = datetime.datetime.strptime(date, '%Y%m%d')
+    begin = enddate - datetime.timedelta(days=period)
+    begin = begin.strftime('%Y%m%d')
+
+    data_all = data.drop('country')\
+                   .select('submission_date_s3',
+                           'client_id',
+                           F.col('histogram_parent_cookie_behavior.4').alias('ETP_on'),
```

Comment:
How will this behave when `4` is not present in the histogram? We could consider adding a test for that.